rpcd/playbooks/roles/rpc_support/defaults/main.yml

```diff
@@ -66,24 +66,19 @@ holland_git_repo_plugins:
 
 holland_pip_wheel_name: holland
 
+holland_packages:
+  - git
+
 holland_venv_enabled: True
 holland_venv_bin: "/openstack/venvs/holland-{{ rpc_release }}/bin"
 holland_venv: "/openstack/venvs/holland-{{ rpc_release }}"
 
 holland_pip_dependencies:
   - MySQL-python
 
-hwraid_apt_repo_url: "http://mirror.rackspace.com/hwraid.le-vert.net/ubuntu/"
-
-hwraid_apt_repos:
-  - { repo: "deb {{ hwraid_apt_repo_url }} {{ ansible_lsb.codename }} main", state: "present" }
-
-hwraid_apt_keys:
-  - { url: "http://mirror.rackspace.com/hwraid.le-vert.net/ubuntu/hwraid.le-vert.net.gpg.key", state: "present" }
-
-hwraid_apt_packages:
-  - megacli
-  - lsiutil
+hwraid_apt_package_urls:
```

Comment:
This is mostly because the hwraid project is not keeping up with releases. 
There is no Xenial dist, so a repo won't work for Xenial. However the Wily packages work for Xenial and for Trusty.